ansible/minigraph/switchlag.xml

```diff
@@ -0,0 +1,916 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```

Comment:
Use the name switch5.xml to align with the current names.
The topology description could be commented in the inventory file.